session9/src/homework/ex7.java

```diff
@@ -0,0 +1,13 @@
+package homework;
+
+import java.time.LocalDate;
+
+public class ex7 {
+    public static void main(String[] args) {
+//        System.out.println("Verify addition:");
+//        int addWeeksToToday = 4;
+//        LocalDate today = LocalDate.now();
+//        today+=addWeeksToToday;
+        //help, please
```

Comment:
Sure! Everything fine until line 10. In order to add weeks to some `LocalDate` object you can use the method `plusWeeks`:
```
LocalDate fourWeeksFromNow = today.plusWeeks(addWeeksToToday);
System.out.println(fourWeeksFromNow);
```